frontend/src/lib/k8s/event.ts

```diff
@@ -20,6 +20,8 @@ export interface KubeEvent {
   [otherProps: string]: any;
 }
 
+export const EVENTS_LIMIT = 2000;
```

Comment:
I'm not sure we should export a const like this. Maybe a static getter/setter for `maxNumEvents` in the Event class, and documented would be better? So users could also easily modify it?

Reply:
updated